tests/playwright/src/specs/volume-smoke.spec.ts

```diff
@@ -77,16 +83,102 @@ describe('Volume workflow verification', async () => {
     await volumeDetails.closeButton.click();
     await playExpect(volumesPage.heading).toBeVisible();
   });
+
+  test('Delete volume from Volumes page', async () => {
+    let volumesPage = await navBar.openVolumes();
+    await playExpect(volumesPage.heading).toBeVisible();
+    let volumeRow = await volumesPage.getVolumeRowByName(volumeName);
+    playExpect(volumeRow).not.toBeUndefined();
+    volumesPage = await volumesPage.deleteVolume(volumeName);
+    await playExpect.poll(async () => await volumesPage.waitForVolumeDelete(volumeName)).toBeTruthy();
+    volumeRow = await volumesPage.getVolumeRowByName(volumeName);
+  });
+
   test('Delete volume through details page', async () => {
+    //re-create a new volume
     let volumesPage = await navBar.openVolumes();
     await playExpect(volumesPage.heading).toBeVisible();
 
-    const volumeRow = await volumesPage.getVolumeRowByName(volumeName);
+    const createVolumePage = await volumesPage.openCreateVolumePage(volumeName);
+    volumesPage = await createVolumePage.createVolume(volumeName);
+
+    await playExpect.poll(async () => await volumesPage.waitForVolumeExists(volumeName)).toBeTruthy();
+
+    //delete it from the details page
+    volumesPage = await navBar.openVolumes();
+    await playExpect(volumesPage.heading).toBeVisible();
+    let volumeRow = await volumesPage.getVolumeRowByName(volumeName);
     playExpect(volumeRow).not.toBeUndefined();
 
     const volumeDetails = await volumesPage.openVolumeDetails(volumeName);
     volumesPage = await volumeDetails.deleteVolume();
 
     await playExpect.poll(async () => await volumesPage.waitForVolumeDelete(volumeName)).toBeTruthy();
+    volumeRow = await volumesPage.getVolumeRowByName(volumeName);
```

Comment:
Same here, shouldnt this just be undefined? Since you already deleted the volume.

Reply:
I think before there was a statement after those lines that checked that the volume did not exist. I deleted them a while ago as you suggested, but I seemed to forget about deleting these ones as well.